javascript-arrays/main.js

```diff
@@ -0,0 +1,31 @@
+var colors = [
+  'red',
+  'white',
+  'blue'
+];
+
+console.log('value of colors[0]:', colors[0]);
+console.log('value of colors[1]:', colors[1]);
+console.log('value of colors[2]:', colors[2]);
+
+console.log('America is ' + colors[0] + ', ' + colors[1] + ', ' + 'and ' + colors[2] + '.');
+
+colors[2] = 'green';
+
+console.log('Mexico is ' + colors[0] + ', ' + colors[1] + ', ' + 'and ' + colors[2] + '.');
+console.log('value of colors', colors);
+var students = [
+  'DJ',
+  'Edwin',
+  'Shawn',
+  'Vic'
+];
+
+var numberOfStudents = students.length;
+console.log('There are 4 students in the classroom');
```

Comment:
You shouldn't be typing the number into the log, you need to use the value of the `numberOfStudents` variable